src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -241,6 +241,12 @@
 WINDOWS_CDB_STACK_OVERFLOW_REGEX = re.compile(
     r'.*ExceptionCode: .*\(Stack overflow\).*')
 
+# Golang specific stack signatures.
+GOLANG_PANIC_REGEX = re.compile(r'^panic: (.*)')
+GOLANG_PANIC_MAX_LEN = 80
```

Comment:
nit: does it make sense `LINE_LENGTH_CAP` instead of this?

Reply:
sounds better, but removed this anyway!